tests/unit/common/etl/transformers/test_cell_expression.py

```diff
@@ -23,3 +24,17 @@ def test_parse_from_metadatas(self):
         expression_rows = parsed[1][1]
         self.assertEqual(expression_table, TableName.EXPRESSION)
         self.assertEqual(expression_rows[0], "635badd5-7d62-4db3-b509-f290a12a1336|ENST00000373020|TPM|92.29\n")
+
+    def test_parse_optimus(self):
+        cell_lines, expression_lines = self.transformer._parse_optimus_bundle(
+            bundle_dir=os.path.abspath("tests/functional/res/etl/d65ef9e6-3fd1-4026-9843-ff573c9e66c8"),
+            test_n_cells=6
```

Comment:
Can you use `mock` here instead of including a test parameter?

Reply:
Modified test data and removed test parameter